src/doc_scan_service/session/create/session.specification.js

```diff
@@ -76,6 +79,11 @@ class SessionSpecification {
     if (identityProfileRequirements) {
       Validation.isPlainObject(identityProfileRequirements, 'identityProfileRequirements');
       this.identityProfileRequirements = identityProfileRequirements;
+
+      if (subject) {
+        Validation.isPlainObject(subject, 'subject');
+        this.subject = subject;
+      }
```

Comment:
Yeah, i get the intent, I did not do that on the Digital-ID side as I left it up to the IDV API to reject.
So, please also extract that subject out of the `identityProfileRequirements` even if it is true that specifying a subject outside of the context of the TFIP would not work.